gradle-modules/gradle-customization/gradle-smithy/gradle.properties

```diff
@@ -0,0 +1,2 @@
+smithyGradleVersion=1.3.0
+smithyJavaVersion=0.0.1
```

Comment:
Any issue with using 0.0.2? It seems available:
https://mvnrepository.com/artifact/software.amazon.smithy.java

Reply:
It's not. Only 0.0.1 is: https://mvnrepository.com/artifact/software.amazon.smithy.java.codegen/plugins

Reply:
Ok - I see.

0.0.2 exists for everything in `software.amazon.smithy.java` but not for anything in `software.amazon.smithy.java.codegen`. I'll fix.

Reply:
But, of course, the two don't seem to be compatible! :( So I'm leaving it on 0.0.1 for now since that works.

Reply:
Huh strange, ok good to know!